ecosystem.md

```diff
@@ -11,19 +11,276 @@ many other data processing projects and frameworks.
 
 <p>Connectors provide code for interfacing with various third-party systems.</p>
 
-<p>Currently these systems are supported:</p>
-
-<ul>
-  <li><a href="{{site.docs-stable}}/dev/connectors/kafka.html" target="_blank">Apache Kafka</a> (sink/source)</li>
-  <li><a href="{{site.docs-stable}}/dev/connectors/elasticsearch.html" target="_blank">Elasticsearch 1.x / 2.x / 5.x / 6.x</a> (sink)</li>
-  <li><a href="{{site.docs-stable}}/dev/connectors/filesystem_sink.html" target="_blank">HDFS</a> (sink)</li>
-  <li><a href="{{site.docs-stable}}/dev/connectors/rabbitmq.html" target="_blank">RabbitMQ</a> (sink/source)</li>
-  <li><a href="{{site.docs-stable}}/dev/connectors/kinesis.html" target="_blank">Amazon Kinesis Streams</a> (sink/source)</li>
-  <li><a href="{{site.docs-stable}}/dev/connectors/twitter.html" target="_blank">Twitter</a> (source)</li>
-  <li><a href="{{site.docs-stable}}/dev/connectors/nifi.html" target="_blank">Apache NiFi</a> (sink/source)</li>
-  <li><a href="{{site.docs-stable}}/dev/connectors/cassandra.html" target="_blank">Apache Cassandra</a> (sink)</li>
-  <li><a href="https://github.com/apache/bahir-flink" target="_blank">Redis, Flume, and ActiveMQ (via Apache Bahir)</a> (sink)</li>
-</ul>
+<p>
+  The following tables list the connectors that are available in the Flink ecosystem.
+  Connector authors are welcome to add their connector to this list via a Github
+  Pull Request. See <a href="{{site.docs-stable}}/improve-website.html">Improve the Website</a>
+  for more details.
+</p>
+
+### Message Queue
+<table class="table table-bordered">
+  <tr>
+    <th>System Name</th>
+    <th>Connector Type</th>
+    <th>Location</th>
+    <th>License</th>
+    <th>Last Released</th>
+    <th>Available For</th>
+    <th>Compatible Flink Versions</th>
+    <th>Maintained By</th>
+  </tr>
+  <tr>
+    <td><a href="{{site.docs-stable}}/dev/connectors/kafka.html" target="_blank">Apache Kafka</a></td>
```

Comment:
should we link here to the Apache Kafka website? 
We can add the link to Flink's documentation to the "Available For" cell. This would allow us to link to the docs of the different APIs (the docs of the Table API Kafka connector are different from the DataStream API Kafka connector docs).

Reply:
I don't think that adding links in "Available For" would mean that we need to have different pages/links for the different APIs. If the links are the same, we can simply link to the same page, IMO.

However, in most cases we have different documentation pages for the different APIs. For example the file system connectors:
* Streaming 
   * [Streaming File Sink](https://ci.apache.org/projects/flink/flink-docs-release-1.8/dev/connectors/streamfile_sink.html)
   * [Monitoring File Source](https://ci.apache.org/projects/flink/flink-docs-release-1.8/dev/datastream_api.html#data-sources) (for streaming)
* Batch
  * [Source](https://ci.apache.org/projects/flink/flink-docs-release-1.8/dev/batch/#data-sources)
  * [Sink](https://ci.apache.org/projects/flink/flink-docs-release-1.8/dev/batch/#data-sinks)
* Table
  * [File System Source](https://ci.apache.org/projects/flink/flink-docs-release-1.8/dev/table/connect.html#file-system-connector)
  * [ORC File Sink](https://ci.apache.org/projects/flink/flink-docs-release-1.8/dev/table/connect.html#csvtablesink)